lib/ansible/modules/commands/psexec.py

```diff
@@ -0,0 +1,527 @@
+    - When setting to C(no), the packets are in plaintext and can be seend by
+      anyone sniffing the network, any process options are included in this.
+    type: bool
+    default: 'yes'
+  connection_timeout:
+    description:
+    - The timeout in seconds to wait when receiving the initial SMB negotiate
+      response from the server.
+    default: 60
+  executable:
+    description:
+    - The executable to run on the Windows host.
+    required: yes
+  arguments:
+    description:
+    - Any arguments as a single string to use when running the executable.
+  working_directory:
+    description:
+    - Changes the working directory set when starting the process.
+    default: C:\Windows\System32
+  asynchronous:
+    description:
+    - Will run the command as a detached process and the module returns
+      immediately after starting the processs while the process continues to
+      run the background.
+    - The I(stdout) and I(stderr) return values will be null when this is set
+      to C(yes).
+    - The I(stdin) option does not work with this type of process.
+    - The I(rc) return value is not set when this is C(yes)
+    type: bool
+    default: 'no'
+  load_profile:
+    description:
+    - Runs the remote command with the user's profile loaded.
+    type: bool
+    default: 'yes'
+  process_username:
+    description:
+    - The user to run the process as.
+    - This can be set to run the process under an Interactive logon of the
+      specified account which bypasses limitations of a Network logon used when
+      this isn't specified.
+    - If ommited then the process is run under the same account as
+      I(connection_username) with a Network logon.
+    - If I(encrypt) is C(no), the username and password are sent as a simple
+      XOR scrambled byte string that is not encrypted. No special tools are
+      required to get the username and password just knowledge of the protocol.
+  process_password:
+    description:
+    - The password for I(process_username).
+    - Required if I(process_username) is defined.
+  elevated:
+    description:
+    - Will run the command with Administrative rights when I(process_username)
+      is defined.
+    - This only affects the process if C(process_username) is set, if it is not
+      then the process is run under the I(connection_username) account and
+      already has an elevated token to the process.
+    - Cannot be C(yes) when I(limited) is also C(yes).
+    type: bool
+    default: 'no'
+  limited:
+    description:
+    - Will run the command with limited rights similar to running a process
+      normally when UAC is turned on.
+    - This only affect the process if C(process_username) is set, if it is not
+      then the process is runder the I(connection_username) account and cannot
+      be run under a limited token.
+    - Cannot be C(yes) when I(elevated) is also C(yes).
+    type: bool
+    default: 'no'
+  use_system:
```

Comment:
Should this just be folded into `process_username`? You'd lose the `required_together` validation, but probably make it up by ditching some of the masking code.